src/main/java/org/mockito/internal/stubbing/answers/ThrowsExceptionClass.java

```diff
@@ -8,14 +8,61 @@
 import org.mockito.internal.exceptions.stacktrace.ConditionalStackTraceFilter;
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
-import org.objenesis.ObjenesisHelper;
+import org.objenesis.Objenesis;
+import org.objenesis.ObjenesisBase;
+import org.objenesis.ObjenesisStd;
+import org.objenesis.instantiator.ObjectInstantiator;
+import org.objenesis.strategy.StdInstantiatorStrategy;
 
 import static org.mockito.internal.exceptions.Reporter.notAnException;
 
 import java.io.Serializable;
+import java.lang.invoke.MethodHandle;
+import java.lang.reflect.Constructor;
 
 public class ThrowsExceptionClass implements Answer<Object>, Serializable {
 
+    private static final Objenesis objenesis;
+    
+    static {
+        final MethodHandle newConstructorForSerialization;
+        final Constructor<Throwable> throwableInit;
+        try {
+            newConstructorForSerialization = MethodHandles.lookup().unreflect(
```

Comment:
@Pr0methean Doesn't the objenesis StdInstantiatorStrategy already provide a  SunReflectionFactoryInstantiator when new InstantiatorOf(Class) is called. It seems to be implemented already, so using new ObjenesisStd() should be fine?